theme/static/js/resourceLandingAjax.js

```diff
@@ -928,9 +928,12 @@ function unzip_irods_file_ajax_submit(res_id, zip_with_rel_path) {
         data: {
             res_id: res_id,
             zip_with_rel_path: zip_with_rel_path,
-            remove_original_zip: "false"
+            remove_original_zip: "false",
+            overwrite: overwrite
         },
         success: function (task) {
+            $('#res_id').val(res_id);
```

Comment:
watch out for id clash
https://github.com/hydroshare/hydroshare/blob/develop/irods_browser_app/templates/irods_upload_add.html#L15